test/libsolidity/interface/FileReader.cpp

```diff
@@ -56,13 +56,16 @@ BOOST_AUTO_TEST_CASE(normalizeCLIPathForVFS_absolute_path)
 		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/b", resolveSymlinks), "/a/b");
 		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/b/", resolveSymlinks), "/a/b/");
 
-		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/./b/", resolveSymlinks), "/a/b/");
-		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/../a/b/", resolveSymlinks), "/a/b/");
 		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/b/c/..", resolveSymlinks), "/a/b");
 		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/b/c/../", resolveSymlinks), "/a/b/");
+		BOOST_CHECK_EQUAL(FileReader::normalizeCLIPathForVFS("/a/./b/", resolveSymlinks), "/a/b/");
+#if !defined(_WIN32) || BOOST_VERSION > 107600
```

Comment:
Do we just want to require latest boost on windows?
Maybe it's overkill for this issue, but then again it's not like windows ships any particular version of boost, so it wouldn't cause too much trouble, would it?

Reply:
I'd be fine with that. Older boost versions in general might slightly affect compiler behavior - though so far all cases I have seen were pretty harmless.

Reply:
So does this stay or does it need to be changed?

Reply:
In this PR we still allow earlier boost versions on Windows versions so we do need it.

I'm going to bump boost version in a separate PR and then I'll remove these.

Reply:
#12024.